.pfnci/schema.yaml

```
@@ -46,7 +46,7 @@ cuda:
   "11.4":
     full_version: "11.4.0"
   "11.5":
-    full_version: "11.5.0"
+    full_version: "11.5.1"
```

Let's pin CI to 11.5.0 as 11.5.1 seems to have bugs in cuSPARSE.

```suggestion
    full_version: "11.5.0"
```

---

I'll regenerate the CI files.